tailwind.config.ts

```diff
@@ -20,6 +20,9 @@ const config: Config = {
       },
       screens: {
         smr: '480px',
+        'md-lg': '1169px', // There was a mysterious break point around 1169px in HeaderClient.tsx,
+        //so a custom breakpoint is create to overwrite it
+        //and now a CSS pattern can be applied with this breakpoint.
```

Comment:
```suggestion
        // There was a mysterious breakpoint around 1169px in `HeaderClient.tsx`,
        // so a custom breakpoint is create to override it and now a CSS pattern
        // can be applied with this breakpoint.
        'md-lg': '1169px',
```

Just some formatting changes

Reply:
sure